src/custom-elements/src/components/carousel/carousel.tsx

```diff
@@ -16,6 +16,14 @@ export class Carousel {
    * To render Carousel with dot controllers
    */
   @Prop() dots: boolean;
+  /**
+   * To render Carousel with pagination indicators
```

Comment:
Change the descriptions and do a `chi build` to generate the new properties in the documentation.

```suggestion
   * To set the autoplay for the carousel
```